iib/workers/tasks/build.py

```diff
@@ -907,17 +912,18 @@ def handle_rm_request(
     with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:
         with set_registry_token(overwrite_from_index_token, from_index_resolved):
             if is_image_fbc(from_index_resolved):
-                err_msg = 'File-Based catalog image type is not supported yet.'
-                log.error(err_msg)
-                raise IIBError(err_msg)
+                opm_registry_rm_fbc(
```

Comment:
could you please log info message that we are working with FBC image.

Reply:
I have added log inside opm_registry_rm_fbc(), would you rather have it added here instead?

Reply:
I know that it was requested by Tim. I thought that we might do this (or similar):
```
is_fbc = is_image_fbc(from_index_resolved)
if is_fbc:
    log.info("Processing File-Based Catalog image")
```
in every `handle_` function. So it is consistent across all API endpoints (ideally the same message, so it is easy to find/grep)
Does it sounds reasonable?

Reply:
Added log here as well :)